fivetran_provider_async/hooks.py

```diff
@@ -102,6 +105,9 @@ async def get_sync_status_async(self, connector_id, previous_completed_at):
         :param previous_completed_at: The last time the connector ran, collected on Sensor
             initialization.
         :type previous_completed_at: pendulum.datetime.DateTime
+        :param reschedule_time: Optional, if connector is in reset state
+            number of seconds to wait before restarting, else Fivetran suggestion used
+        :type reschedule_time: int
```

Comment:
nitpick: is this kind of type annotation newly introduced in this PR? I just checked other files and could not find similar anntation.

Reply:
@Lee-W this file already had a similar annotation: https://github.com/astronomer/airflow-provider-fivetran-async/blob/16b919341e5c476ac73a27ff5e09f90babd495ea/fivetran_provider_async/hooks.py#L27

We can change the annotation for this project in a separate PR.

Reply:
sounds good 👍